src/main/java/duke/dukeexception/DukeIncorrectTimeFormatException.java

```diff
@@ -0,0 +1,14 @@
+package duke.dukeexception;
+
+public class DukeIncorrectTimeFormatException extends DukeException {
+
+    public DukeIncorrectTimeFormatException(String message){
```

Comment:
Perhaps leave a space after the argument closing parenthesis for better readability? I noticed the same issue in several other places too...

```suggestion
    public DukeIncorrectTimeFormatException(String message) {
```